app/src/main/java/ru/chaichuk/hwapp/HWApp.kt

```diff
@@ -0,0 +1,11 @@
+package ru.chaichuk.hwapp
+
+import android.app.Application
+
+class HWApp : Application() {
+
+    companion object {
```

Comment:
better create another file with name constants or config
but it's up to you

Reply:
I used it to get global app context. So leaved it for global constants. Next time I will move them to external config file